static/templates/previewable_url_tooltip.hbs

```diff
@@ -0,0 +1,33 @@
+<style>
+    .previewable-container {
+        display: flex;
+    }
+    .previewable-container > img {
+        padding: 2px 5px 0 0;
+        width: 1rem;
+        height: 1rem;
+    }
+    .previewable-container > div {
+        min-width: 0;
+    }
+    .previewable-title, .previewable-details {
+        display: block;
+        max-width: 100%;
+        overflow: hidden;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+    }
+    .previewable-title {
+        font-size:medium;
+    }
+    .previewable-details {
+        color: gray;
+    }
+</style>
```

Comment:
Let's put these styles as a section inside `rendered_markdown.css`; we don't do inline styles inside template files as a coding pattern in this project.